match-upstream.md

```diff
@@ -6,26 +6,29 @@ Public API: the main public API of zxcvbn-php does not need to change with this
 
 Planned Changes:
 * Zxcvbn: I updated `passwordStrength()` to more closely mimic upstream's `main.coffee`. I arguably went too far, and might walk that back a bit.
-* Feedback: Added.
+* Feedback: added.
   * [ ] Need to flesh out based on upstream. Should be pretty straightforward.
+  * [ ] Write tests for the feedback.
 * Time estimator: added.
   * [x] Just need to flesh out `displayTime()` based on upstream. Should be straightforward.
-* Matchers: 
-  * The structure here is the same as upstream. :-)
-  * Individual matchers might or might not match; the best way to find out is with unit tests.
-  * [ ] They have a `reverse_dictionary_match` which we don't have. Should be somewhat straightforward as a subclass of `DictionaryMatch`.
-  * They have a `regex_match` which seems to be a pure equivalent of `YearMatch`
-  * We have `DigitMatch` which they don't. I guess we should get rid of it.
-* [ ] `Scorer`:  This is vastly different between upstream and this port. Upstream's algorithm is complicated and hard to follow. I think this will be the hardest thing to bring up to parity. *Some* of this may be similar to `Searcher::getMinimumEntropy()` but I really can't tell.
-  * Some of the other language ports e.g. https://github.com/rianhunter/zxcvbn-cpp/blob/zxcvbn-cpp/native-src/zxcvbn/scoring.cpp may also be useful references when porting.
-* :question: `ScorerInterface`: In upstream, `scoring.most_guessable_match_sequence` returns a hash with password/guesses/guesses_log10/sequence. Our current `ScorerInterface` has methods for `getScore()` and `getMetrics()`. Our interface is clearly "cleaner", but it might make more sense to just mirror upstream. :neutral_face:
+  * [ ] Write tests for the time estimator.
+* Matchers: added.
+  * [x] The majority of the matchers have now been ported.
+  * [x] Port the tests for the matchers.
+  * [ ] RepeatMatch: `base_guesses` and `base_matches` are still missing, but this will require the `Scorer` to be up and running before we can implement them.
+* Entropy: this is a feature unique to the PHP library.
```

Comment:
Entropy used to be part of zxcvbn, and was removed in 4.0.1. See the [release notes](https://github.com/dropbox/zxcvbn/releases/tag/4.0.1) for more details on why. I don't think we should keep it in the PHP library.